app/models/concerns/offeror.rb

```diff
@@ -0,0 +1,99 @@
+module Offeror
+
+	def self.import(file, current, comment)
```

Comment:
Agree. I've extracted two methods from the original one, but it can be refactored more thoroughly.
Can we create a separate GitHub issue for this task?